bazel/patches/absl.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/absl/base/BUILD.bazel b/absl/base/BUILD.bazel
```

Comment:
I cannot find a better way to work around this. Inside google we have:
```
        "//tools/cc_target_os:windows": [],
        # absl:google3-begin(emscripten is google3 only)
        "//tools/cc_target_os:emscripten": [],
        # absl:google3-end
        "//conditions:default": ["-pthread"],
```

We probably need to add the same options to upstream absl. @lizan @jplevyak @PiotrSikora Any suggestion?

Reply:
Created https://github.com/abseil/abseil-cpp/issues/693 to track this.

Reply:
Yeah we definitely need a right select there, but currently bazel doesn't define a common cpu/platform for wasm. I raised PR https://github.com/bazelbuild/platforms/pull/12 but not merged yet.